fw/Core/Hitcon/Logic/Display/test-display.cc

```diff
@@ -9,7 +9,7 @@ void print_buf(int frame, uint8_t *buf) {
   printf("frame %d\n", frame);
   for (int y = 0; y < DISPLAY_HEIGHT; y++) {
     for (int x = 0; x < DISPLAY_WIDTH; x++) {
-      uint8_t ch = buf[y * DISPLAY_WIDTH + x];
+      uint8_t ch = !!(buf[x] & (1 << y));
```

Comment:
Maybe add a macro `DISPLAY_GET` since we already have `DISPLAY_SET` and `DISPLAY_UNSET`? (also I would suggest capitalize `DISPLAY_render_char` since it seems weird)